src/java.sql/share/classes/java/sql/Statement.java

```diff
@@ -32,7 +32,7 @@
  * <P>The object used for executing a static SQL statement
  * and returning the results it produces.
  * <P>
- * By default, only one <code>ResultSet</code> object per <code>Statement</code>
+ * By default, only one {@code ResultSe} object per <code>Statement</code>
```

Comment:
Now, after having started review, that button no longer show, can only add comment

Reply:
I see I can "finish your review" (2)

Can you, the author, do the same?

Reply:
Looking now